src/Microsoft.ML.StaticPipe/LpNormalizerStaticExtensions.cs

```diff
@@ -47,16 +47,16 @@ public Reconciler(LpNormalizingEstimatorBase.NormFunction normKind, bool ensureZ
                 foreach (var outCol in toOutput)
                     pairs.Add((outputNames[outCol], inputNames[((OutPipelineColumn)outCol).Input]));
 
-                return new LpNormalizingEstimator(env, pairs.ToArray(), _norm, _ensureZeroMean);
+                return new LpNormNormalizingEstimator(env, pairs.ToArray(), _norm, _ensureZeroMean);
             }
         }
 
         /// <include file='../Microsoft.ML.Transforms/doc.xml' path='doc/members/member[@name="LpNormalize"]/*'/>
         /// <param name="input">The column to apply to.</param>
```

Comment:
>The column to apply to. [](start = 32, length = 23)

Isn't this a row?

Reply:
Fixed. Thanks.

---
In reply to: [265340830](https://github.com/dotnet/machinelearning/pull/2888#discussion_r265340830) [](ancestors = 265340830)